charts/fission-all/templates/timer/deployment.yaml

```diff
@@ -38,6 +38,9 @@ spec:
       {{- if .Values.priorityClassName }}
       priorityClassName: {{ .Values.priorityClassName }}
       {{- end }}
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets: {{- toYaml . | nindent 8 }}
```

Comment:
I think imagePullSecrets values wont work. As its array we need to have it on new line with indentation.
Please see https://github.com/prometheus-community/helm-charts/blob/b2b9606c293ddfa129789f33ee66b8437b557807/charts/alertmanager/templates/statefulset.yaml#L32
for example.

Eg. https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/pods/private-reg-pod.yaml